tests/functional/spec/features/repeating_blocks/list_collector_repeating_blocks.spec.js

```diff
@@ -52,13 +53,13 @@ async function addCompany(
 }
 
 describe("List Collector Repeating Blocks", () => {
-  describe("Given a normal journey through the list collector with repeating blocks, the answers can be submitted.", () => {
+  describe("Given a normal journey through the list collector with repeating blocks", () => {
```

Comment:
Minor - whats the convention for `describe` naming?
I find it a bit strange that the describe is `Given a normal journey.` without any kind of when, then
I'd favour changing it to just `describe('A normal journey...')`

Reply:
ahhh, I see, that makes more sense to me now - the latter ones with Given a journey..., any incomplete... don't fit this style but I think that works if they're made consistent with these ones